client/galaxy/scripts/layout/page.js

```diff
@@ -22,7 +22,12 @@ const View = Backbone.View.extend({
             show_inactivity_warning: false,
             inactivity_box_content: "",
             hide_panels: false,
-            hide_masthead: false
+            hide_masthead: false,
+            masthead_show_analysis: true,
+            masthead_show_workflows: true,
+            masthead_show_workflows: true,
+            masthead_show_activities: false,
+            masthead_show_scratchbook: true,
```

Comment:
If we would introduce `hide_masthead_analysis` instead the default would be false per definition and most of these values would not have to be initialized/specified when building the masthead. Also the naming would be more consistent with the parameters above.

Reply:
The problem with a meta property like that is that we've discussed per role display of features in the UI. Might be better to just break everything down now? Then we can do ``view_default && role_default`` for each option? But maybe that could be pushed down a layer? All this code will be obsoleted by #9071 - I'll revisit this and try both after that is integrated and I redo this.